priv/templates/install/beacon_router_scope.ex

```diff
@@ -0,0 +1,7 @@
+  import Beacon.Router
```

Comment:
```suggestion

  import Beacon.Router
```

So the generated file is left formatted.